atomium/templates/table/table.component.inc

```diff
@@ -79,89 +81,141 @@ function atomium_preprocess_table(array &$variables, $hook) {
     );
   }
 
-  $ts = \count($header) ? tablesort_init($header[0]) : array();
+  $ts = !empty($header) ? tablesort_init($header[0]) : array();
 
   // Format the table colgroups row.
-  $variables['colgroups'] = _atomium_table_format_rows('colgroup', array($variables['colgroups']), $header, $ts, FALSE, $variables);
+  $variables['colgroups'] = _atomium_table_format_rows(
+    'colgroup',
+    array($variables['colgroups']),
+    $header,
+    $ts,
+    FALSE,
+    $variables
+  );
 
   // Format the table header rows.
-  $variables['header'] = _atomium_table_format_rows('header', $variables['header'], $header, $ts, TRUE, $variables);
+  $variables['header'] = _atomium_table_format_rows(
+    'header',
+    $variables['header'],
+    $header,
+    $ts,
+    TRUE,
+    $variables
+  );
 
   // Format the table footer rows.
-  $variables['rows'] = _atomium_table_format_rows('body', $variables['rows'], $header, $ts, FALSE, $variables);
+  $variables['rows'] = _atomium_table_format_rows(
+    'body',
+    $variables['rows'],
+    $header,
+    $ts,
+    FALSE,
+    $variables
```

Comment:
As we follow Drupal CS, here is missing a comma. Check [here](https://www.drupal.org/docs/develop/standards/coding-standards#array).

Reply:
It's a function call, adding a comma there will trigger a syntax error :-)